package.json

```diff
@@ -152,7 +152,7 @@
   },
   "version": "4.0.0",
   "dependencies": {
-    "@popperjs/core": "^2.5.4",
+    "@popperjs/core": "^2.7.2",
```

Comment:
There's no real impetus behind this upgrade other than that it needs to be done sometime.